### ISSUE TITLE
Karma and Todos for Entony Lekhtman

### ISSUE THREAD

**Issue by zvili** [TechnionYP5777/Bugquery#6]
_From @tonylekhtman on October 29, 2016 10:29_

- [x]  Use a picture, photo or distinct avatar. Don''t use an automatic icon:
- [x]  Make sure you present yourself by a readable **nick**
- [x]  Describe yourself in the bio page; FYI, here is my description:
- [x]  State that you are (not) in search for a job, i.e., mark, or unmark this box:

_Copied from original issue: SpartanRefactoring/Spartanizer#579_

**Comment by zvili** [comment]
_From @tonylekhtman on October 29, 2016 10:41_

@YossiGil: Done.

**Comment by zvili** [comment]
_From @RonGatenio on October 29, 2016 20:20_

Nice page Tony!
Here is mine #584

**Comment by zvili** [comment]
_From @AmitOhayon on October 31, 2016 13:25_

This Karma page represents everything that I love about you.
What do you think of mine? #680

**Comment by yossigil** [comment]
Can you summarize here all issues you dealt with towards Sprint 1?

@AdiOmari

**Comment by tonylekhtman** [comment]
[x] learned SQL.
[x] learned how to work with SQL in Java.(Jdbc)
[x] Imported dummy xml of the posts to a local mysql server with @ZivIzhar.
[x] Implemented the import of the xml and creation of the mysql server in java using jdbc with @ZivIzhar .

**Comment by yossigil** [comment]
List of **ISSUES**.

**Comment by tonylekhtman** [comment]
issues: #9 #16

**Comment by yossigil** [comment]
Please remove the blurb in the comment before and leave the previous one only.

**Comment by tonylekhtman** [comment]
Done
